pkgs/servers/web-apps/freshrss/default.nix

```diff
@@ -13,6 +19,8 @@ stdenvNoCC.mkDerivation rec {
 
   passthru.tests = nixosTests.freshrss;
 
+  buildInputs = [ php ];
```

Comment:
is it really wanted?

Reply:
without this `patchShebangs` doesn't know about php, and so it leaves the shebangs as `#!/usr/bin/env php`. AFAICT`patchShebangs` just crawls the PATH, so this `buildInputs` could be removed if we invoke `patchShebangs` with `PATH` like this:

```sh
PATH="${php}/bin:$PATH" patchShebangs cli/*.php app/actualize_script.php
```

i only see this pattern in a few places: any idea what the best conventions are here?

Reply:
oh thanks for the explanation. Did not know that. Maybe your solution is fine enough, because in the next step when using it php is required.